### PR TITLE
Database

### PR DESCRIPTION
**Pull request by avscott562**

### CODE REVIEW

File: db/schema.sql [hunk 1]
```diff
@@ -1,5 +1 @@
-DROP DATABASE IF EXISTS exampledb;
-CREATE DATABASE exampledb;
-
-DROP DATABASE IF EXISTS testdb;
-CREATE DATABASE testdb;
+CREATE DATABASE measurements_db;
```
Comment (SirHexxus) [inline db/schema.sql]: Cannot Create Databases in JawsDB, we need to use the `zh5j2ywf78gnkyey` database.

File: models/measurement.js [hunk 2]
```diff
@@ -0,0 +1,217 @@
+module.exports = function(sequelize, DataTypes) {
```
Comment (SirHexxus) [inline models/measurement.js]: This is good, but we need to make sure that we're storing and collecting the same data. I'll put together a list of the data that I am collecting on the front-end, so that we can make some decisions about which data we're using.

File: models/user.js [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+      },
+      phoneNumber: {
+        type: DataTypes.STRING
+      },
+      colorPreferences: {
+        type: DataTypes.TEXT
+      },
+      project: {
+        type: DataTypes.STRING
+      },
```
Comment (SirHexxus) [inline models/user.js]: Line 20-28, I think this data needs to be associated directly with each measurement set, rather than for each user.

File: config/config.js [hunk 4]
```diff
@@ -1,5 +1,7 @@
 require("dotenv").config();
 
+// CODE BELOW DOESN'T GIVE ME "USE_ENV_VARIABLE" ERROR on index.js.  However, can't connect to heroku database since I don't have the Heroku app name (funny name like wild lakes 1234).  I tried using the https://x-caliber.herokuapp.com/ that James slacked out but that didn't work.  Could be because it's not my heroku.  Not sure.  Not sure if this is working since I can't test it.
```
Comment (SirHexxus) [inline config/config.js]: JawsDB info:

Host: kavfu5f7pido12mr.cbetxkdyhwsb.us-east-1.rds.amazonaws.com
Username: qvgba9tchdssplsm 
Password: byq32sz41enmxb7m 
Port: 3306 
Database: zh5j2ywf78gnkyey

File: server.js [hunk 5]
```diff
@@ -34,7 +34,7 @@ if (process.env.NODE_ENV === "test") {
 }
 
 // Starting the server, syncing our models ------------------------------------/
-db.sequelize.sync(syncOptions).then(function() {
+db.sequelize.sync({ force: true }).then(function() {
```
Comment (SirHexxus) [inline server.js]: Why this change? Do we need to `force: true`?

Comment (avscott562) [reply server.js]: I thought we were supposed to use force true on the database sync.  I can remove it.

File: config/config.js (hunk 4 above)
Comment (avscott562) [reply config/config.js]: I put the JawsDB info in the dotenv file so that we we wouldn't share database passwords on Github.  I believe that was a project requirement.

File: models/user.js (hunk 3 above)
Comment (avscott562) [reply models/user.js]: For MVP, I thought in the user table would be better.  Eventually, I think it should have its own table as a user could eventually have more than one project.  I don't see that a user would have more than one measurement set though.  I don't mind moving it to the measurements table though.

File: db/schema.sql (hunk 1 above)
Comment (avscott562) [reply db/schema.sql]: I was using the hero.sql for that but I'll move it to the schema.sql.  I'll delete the hero.sql.
